CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 All notable changes to `dash` will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/).
 
+## [2.10.3] - 2023-xx-yy
```

Comment:
```suggestion
## [UNRELEASED]
```
Will probably be 2.11.0 given the other things about to merge, but best to leave with no number until it's actually in a release regardless.